packages/flutter/lib/src/rendering/flex.dart

```diff
@@ -198,7 +197,7 @@ enum CrossAxisAlignment {
   baseline,
 }
 
-bool/*?*/ _startIsTopLeft(Axis direction, TextDirection textDirection, VerticalDirection verticalDirection) {
+bool? _startIsTopLeft(Axis direction, TextDirection? textDirection, VerticalDirection verticalDirection) {
```

Comment:
verticalDirection should be nullable

Reply:
to clarify, it doesn't really matter, this is just for symmetry

Reply:
Done